plugins/modules/idrac_certificates.py

```diff
@@ -399,6 +421,26 @@ def get_cert_url(actions, op, certype, res_id):
     return dynurl
 
 
+def upload_ssl_key(module, idrac, actions, ssl_key, res_id):
+    if not (os.path.exists(ssl_key) or os.path.isdir(ssl_key)):
+        module.exit_json(msg="Unable to locate the SSL key file at {0}.".format(ssl_key), failed=True)
+    try:
+        with open(ssl_key, "r") as scert:
+            scert_file = scert.read()
+    except OSError as file_err:
+        module.exit_json(msg=str(file_err), failed=True)
+    if not module.check_mode:
+        upload_url = actions.get("#DelliDRACCardService.UploadSSLKey")
```

Comment:
handle case when no upload key url is found - module.exit_json("UPload SSL key not supported")

Reply:
Done